ledger/ledgercore/totals.go

```diff
@@ -52,6 +53,15 @@ type AccountTotals struct {
 	RewardsLevel uint64 `codec:"rwdlvl"`
 }
 
+// OnlineRoundParamsData keeps track of parameters needed for agreement from maxBalLookback ago
+type OnlineRoundParamsData struct {
+	_struct struct{} `codec:",omitempty,omitemptyarray"`
+
+	OnlineSupply    uint64                    `codec:"a"`
```

Comment:
the alphabetical single letter msgp field name thing — while certainly the optimal smallest-sized key name, we are only storing a few hundred rounds of OnlineRoundParamsData..

Reply:
done